test/unit/init/cli/YargsCliExtractor.test.ts

```diff
@@ -52,6 +52,14 @@ describe('A YargsCliExtractor', (): void => {
     expect(error).toHaveBeenCalledWith('Unknown argument: unsupported');
   });
 
+  it('can error when multiple values are provided for a non array type parameter.', async(): Promise<void> => {
+    extractor = new YargsCliExtractor(parameters, { strictMode: true });
+    const argv = [ 'node', 'script', '-p', '3000', '-b', 'http://localhost:3000/', '-p', '3001' ];
```

Comment:
Can you also add an entry with multiple values that is legal? Just to make sure that we never break that one accidentally.